client/src/app/site/pages/meetings/pages/participants/participants.subscription.ts

```diff
@@ -17,17 +17,13 @@ export const getParticipantVoteInfoSubscriptionConfig: SubscriptionConfigGenerat
         ids: [id],
         follow: [
             {
-                idField: `user_ids`,
+                idField: `meeting_user_ids`,
                 fieldset: [
                     `default_vote_weight`,
                     `is_physical_person`,
                     `is_active`,
-                    `meeting_ids`,
-                    { templateField: `vote_delegated_$_to_id` },
-                    { templateField: `vote_delegations_$_from_ids` },
-                    { templateField: `vote_weight_$` },
-                    { templateField: `group_$_ids` },
-                    `is_present_in_meeting_ids`
+                    `is_present_in_meeting_ids`,
+                    `group_ids`
```

Comment:
You are subscribing to the meeting users, but you seem to be requesting user-fields. Meeting users don't have a `default_vote_weight` property. Also: Where did the `vote_weight`, `vote_delegations_from_ids` and `vote_delegated_to` go?

Please use the meeting user fields in this subscriptions and then _follow_ the `user_id` property of the meeting user to get the user fields